_preliminary/sweep_over_habitat_to_get_prevalence.py

```diff
@@ -0,0 +1,83 @@
+from functools import partial
+
+from emodpy.emod_task import EMODTask
+from idmtools.builders import SimulationBuilder
+from idmtools.core.platform_factory import Platform
+from idmtools.entities.experiment import Experiment
+
+from run_sims import manifest
+from run_sims.build_config import set_full_config
+from run_sims.other import include_post_processing
```

Comment:
run_sims.other is not checked in.

Reply:
I renamed this file to "helpers", and have changed the code accordingly